.autover/changes/243ac697-e410-4abf-b697-7c05e18bdda4.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Projects": [
+    {
+      "Name": "Amazon.AspNetCore.DataProtection.SSM",
+      "Type": "Patch",
```

Comment:
should be minor?

Reply:
I went with patch because we are doing this release to pull in the latest bug fixes in the SDK. Users of this library will not experience any new features.